.github/workflows/ci_policies.yml

```
@@ -0,0 +1,56 @@
+name: ci-policies
+on:
+  workflow_call:
+env:
+  GO_VERSION: '1.22'
+
+jobs:
+  cerbosCheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)